packages/hash/frontend/src/components/hooks/useUser.ts

```diff
@@ -45,5 +71,10 @@ export const useUser = (options?: Omit<QueryHookOptions, "errorPolicy">) => {
   //   client.writeQuery();
   // };
 
-  return { user, refetch, loading };
+  return {
+    user,
+    kratosSession,
+    refetch: () => Promise.all([refetchUser]),
```

Comment:
Why do we need this Promise.all?

Reply:
Because I forgot to add a second value to the array 😅
```suggestion
    refetch: () => Promise.all([refetchUser, fetchKratosIdentity]),
```
Thanks for spotting, have fixed in https://github.com/hashintel/hash/pull/889/commits/1b2a204df4e694605d95036daa88efe2959862c8